source/PyMaterialX/PyMaterialXCore/CMakeLists.txt

```diff
@@ -13,9 +13,8 @@ set_target_properties(
     OUTPUT_NAME PyMaterialXCore
     COMPILE_FLAGS "${EXTERNAL_COMPILE_FLAGS}"
     LINK_FLAGS "${EXTERNAL_LINK_FLAGS}"
+    # This is suspicious...
```

Comment:
What are your thoughts on this, @JeanChristopheMorinPerso?  This was added by @pmolodo to enable shared libraries on MacOS and Linux, and I want to make sure it's aligned with these latest updates to Python distribution.

Reply:
If I remember correctly, I wrote this because it makes the wheel a little bit less "self-contained".

```
[jcmorin@arch01 asd]$ readelf -d MaterialX/PyMaterialXCore.cpython-311-x86_64-linux-gnu.so

Dynamic section at offset 0x238a58 contains 30 entries:
  Tag        Type                         Name/Value
 0x0000000000000001 (NEEDED)             Shared library: [libdl.so.2]
 0x0000000000000001 (NEEDED)             Shared library: [libstdc++.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libm.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libgcc_s.so.1]
 0x0000000000000001 (NEEDED)             Shared library: [libc.so.6]
 0x000000000000000e (SONAME)             Library soname: [PyMaterialXCore.cpython-311-x86_64-linux-gnu.so]
 0x000000000000000f (RPATH)              Library rpath: [$ORIGIN/../../lib:$ORIGIN:/tmp/tmp5iogfed3/build/installed/lib]
```
This is is coming from one of the Linux wheels. Preferably, only `$ORIGIN` would be there. We don't want anything outside the installed wheel to interfere with it. I'll see if I can conditionally add the RPATH for non wheel builds.

Reply:
Alright, I pushed a commit that should fix this.